scripts/buildkite/gocov.sh

```diff
@@ -8,10 +8,6 @@ go get github.com/dmetzgar/goveralls
 # download cover files from all the tests
 mkdir -p build/coverage
 buildkite-agent artifact download "build/coverage/unit_cover.out" . --step ":golang: unit test" --build "$BUILDKITE_BUILD_ID"
-buildkite-agent artifact download "build/coverage/integ_cassandra_cover.out" . --step ":golang: integration test with cassandra" --build "$BUILDKITE_BUILD_ID"
```

Comment:
are those useless? I assume we don't generate code coverage in integration tests

Reply:
Yeah I believe so. 
And if you look at the list, it doesn't take ndc into account. I don't have permission, would you have help me kick off a run to see how the coverage would change?